src/components/Reports/SalaryReport/SalaryReportTable.tsx

```diff
@@ -1,10 +1,285 @@
-import React from 'react';
-import { Box } from '@material-ui/core';
+/* eslint-disable import/no-unresolved */
+import React, { ChangeEvent, useEffect, useState } from 'react';
+import _ from 'lodash';
+import clsx from 'clsx';
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  colors,
+  Grid,
+  SvgIcon,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TablePagination,
+  TableRow,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import CodeIcon from '@material-ui/icons/Code';
+import PrintIcon from '@material-ui/icons/Print';
+import { CSVLink } from 'react-csv';
+import { DateTime } from 'luxon';
+import { useTranslation } from 'react-i18next';
+import {
+  FourteenMonthReportContact,
+  FourteenMonthReportCurrencyType,
+  // eslint-disable-next-line import/extensions
+} from 'graphql/types.generated';
+// eslint-disable-next-line import/extensions
```

Comment:
@OzzieOrca Yeah, maybe we need to include `ts` files in the `tsconfig` since using `baseUrl` doesn't know the 'multiple dotted' extension file. I will have a look.